segmentation violation occurs at ecsher logs command

```
ecsher logs task --name 0d005eaa6910412db0458a3ccbb4145d  
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x16b13f2]

goroutine 1 [running]:
github.com/Mic-U/ecsher/cmd.getTaskLogs()
        /home/runner/work/ecsher/ecsher/cmd/logs.go:155 +0x412
github.com/Mic-U/ecsher/cmd.glob..func6(0x1c0ed20, 0xc00019cd20, 0x1, 0x3)
        /home/runner/work/ecsher/ecsher/cmd/logs.go:51 +0x95
github.com/spf13/cobra.(*Command).execute(0x1c0ed20, 0xc00019ccf0, 0x3, 0x3, 0x1c0ed20, 0xc00019ccf0)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:856 +0x2c2
github.com/spf13/cobra.(*Command).ExecuteC(0x1c0efa0, 0x101a501, 0x0, 0x0)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
github.com/Mic-U/ecsher/cmd.Execute()
        /home/runner/work/ecsher/ecsher/cmd/root.go:38 +0x2d
main.main()
        /home/runner/work/ecsher/ecsher/main.go:21 +0x25
```

--- a/cmd/logs.go
+++ b/cmd/logs.go
@@ -152,7 +152,7 @@ func getTaskLogs() {
 		container = taskDefinition.ContainerDefinitions[0]
 	}
 
-	if container.LogConfiguration.LogDriver != "awslogs" {
+	if !util.IsAwslogsLogDriver(container) {
 		fmt.Println("logDriver must be awslogs")
 		os.Exit(1)
 	}

--- a/util/logs.go
+++ b/util/logs.go
@@ -9,6 +9,7 @@ import (
 )
 
 const (
+	Awslogs            = "awslogs"
 	LogGroupKey        = "awslogs-group"
 	LogRegionKey       = "awslogs-region"
 	LogStreamPrefixKey = "awslogs-stream-prefix"
@@ -55,4 +56,11 @@ func AscendingSortTaskLogs(logs []cloudwatchTypes.OutputLogEvent) []cloudwatchTy
 		return *logs[i].Timestamp < *logs[j].Timestamp
 	})
 	return logs
+}
+
+func IsAwslogsLogDriver(containerDefinition types.ContainerDefinition) bool {
+	if containerDefinition.LogConfiguration == nil {
+		return false
+	}
+	return containerDefinition.LogConfiguration.LogDriver == Awslogs
 }

--- a/util/logs_test.go
+++ b/util/logs_test.go
@@ -102,3 +102,46 @@ func TestAscendingTaskLogs(t *testing.T) {
 		}
 	}
 }
+
+func TestIsAwslogsLogDriver(t *testing.T) {
+	cases := []struct {
+		container types.ContainerDefinition
+		expected  bool
+	}{
+		{
+			container: types.ContainerDefinition{
+				LogConfiguration: nil,
+			},
+			expected: false,
+		},
+		{
+			container: types.ContainerDefinition{
+				LogConfiguration: &types.LogConfiguration{},
+			},
+			expected: false,
+		},
+		{
+			container: types.ContainerDefinition{
+				LogConfiguration: &types.LogConfiguration{
+					LogDriver: "json",
+				},
+			},
+			expected: false,
+		},
+		{
+			container: types.ContainerDefinition{
+				LogConfiguration: &types.LogConfiguration{
+					LogDriver: "awslogs",
+				},
+			},
+			expected: true,
+		},
+	}
+
+	for _, c := range cases {
+		actual := IsAwslogsLogDriver(c.container)
+		if actual != c.expected {
+			t.Errorf("expect %v, got %v", c.expected, actual)
+		}
+	}
+}

